packages/node/src/utils/flush-chunks.ts

```diff
@@ -82,15 +109,18 @@ const processChunk = async (chunk, shareMap, hostStats) => {
   }
 
   try {
+    console.log('remote', knownRemotes);
     // Extract the remote name from the URL
     //@ts-ignore
     const remoteName = new URL(
+      //@ts-ignore
       globalThis.__remote_scope__._config[remote],
     ).pathname
       .split('/')
       .pop();
 
     // Construct the stats file URL from the remote config
+    //@ts-ignore
```

Comment:
more work needed here, but there are other isssues beyond this that must be accounted for.